cmd/soroban-cli/src/utils/contract-init-template/contracts/hello_world/Cargo.toml.removeextension

```diff
@@ -15,4 +15,4 @@ doctest = false
 soroban-sdk = { workspace = true }
 
 [dev_dependencies]
```

Comment:
Since we are here, this should also be updated. `cargo add --dev ...` will error if there is an `_`.
```suggestion
[dev-dependencies]
```